src/formatDate.js

```diff
@@ -50,7 +50,61 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const formated = [];
+  const splitDate = date.split(fromFormat[3]);
+  let year;
+  let month;
+  let day;
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    if (fromFormat[i] === 'YYYY' || fromFormat[i] === 'YY') {
+      year = splitDate[i];
+    }
+
+    if (fromFormat[i] === 'MM') {
+      month = splitDate[i];
+    }
+
+    if (fromFormat[i] === 'DD') {
+      day = splitDate[i];
+    }
```

Comment:
- please use switch case statement instead of multiple if blocks 
- YYYY, MM, DD are magic words, so it's make sense to move them to constant